mhs/common/mhs_common/errors/ebxml_handler.py

```diff
@@ -0,0 +1,51 @@
+from typing import Tuple, AnyStr, Dict, Optional
+from xml.etree import ElementTree as ET
+
+from utilities.integration_adaptors_logger import IntegrationAdaptorsLogger
+
+from mhs_common.messages.ebxml_error_envelope import EbxmlErrorEnvelope
+
+logger = IntegrationAdaptorsLogger('EBXML_ERROR_HANDLER')
+
+ERROR_RESPONSE_DEFAULTS = {
+    'errorType': 'ebxml_error'
+}
+
+
+def handle_ebxml_error(code: int, headers: Dict, body: str) -> Tuple[int, Optional[AnyStr]]:
+    """
+    Analyzes response from NHS and returns result of interpretation to external client
```

Comment:
```suggestion
    Analyzes response from MHS and returns result of interpretation to external client
```

And docstring could possibly explain a bit more about the error handling, or just say that it's checking for an ebxml error.

Reply:
Docstring amended.